static/docs/commands-reference/import-url.md

```diff
@@ -151,18 +149,25 @@ the _Add Files_ step mentioned before.
 
 ## Example: Tracking a remote file
 
-The _Get Started_ section demonstrates a simple pipeline. In the
-[Add Files](/doc/get-started/add-files) step we are shown how to download a
-file, then use `dvc add` to integrate it with the workspace.
-
 An advanced alternate to initialize the _Get Started_ workspace, is using
 `dvc import-url`:
 
 ```dvc
 $ dvc import-url https://dvc.org/s3/get-started/data.xml data/data.xml
 Importing 'https://dvc.org/s3/get-started/data.xml' -> 'data/data.xml'
 [##############################] 100% data.xml
-...
+[##############################] 100% data.xml
+
+Adding 'data/data.xml' to 'data/.gitignore'.
+
+WARNING: data 'data/data.xml' exists. Removing before checkout.
```

Comment:
@efiop is not fixed yet?

Reply:
Not yet fixed. We are working on it in https://github.com/iterative/dvc/pull/2358. Let's not include this into the command-reference, since it is a bug.

Reply:
Removed in 0f50522